kolena/_utils/dataframes/transformers.py

```diff
@@ -51,19 +54,97 @@ def _normalize_dicts(dict_list: List[Dict[str, Any]], sep: str = ".", max_level:
     # Get all unique keys across all dictionaries
     all_keys = set().union(*flattened_dicts)
 
-    # Normalize all dictionaries to have all keys, filling missing ones with None
+    # Normalize all dictionaries to have all keys,
+    # filling missing ones with None and parsing existing ones
     for d in flattened_dicts:
         for key in all_keys:
-            if key not in d:
-                d[key] = None
+            d[key] = _try_parse(d.get(key, None))
```

Comment:
what's the rational to add parse at here? `_normalize_dicts` is only used by `json_normalize`.